shop/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**/target/
+!**/src/test/**/target/
+
+### STS ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+
+### IntelliJ IDEA ###
+.idea
```

Comment:
This is the critical problem that I told you about, if you look to the files that you try to push to master you'll see the `.idea` folder which contains the libraries which you are using. We **never ever** do this. No matter what package manager we have we never push them on the remote repository, also the IntelliJ local settings. The flow is that the **pom.xml** specifiec which dependencies this project has and they are downloaded locally by each ran project. You should add "**/.idea" in this `.gitignore` folder and we also need to remove the folder from the tracked ones: `git rm -r --cached path_to_your_folder/`

Reply:
your help did the job for me here :)